pkg/apis/autoscaling/register.go

```diff
@@ -216,6 +216,9 @@ const (
 	// PanicThresholdPercentageMax is the counterpart to the PanicThresholdPercentageMin
 	// but bounding from above.
 	PanicThresholdPercentageMax = 1000.0
+
+	// ProgressDeadlineSeconds is the label key for the per revision progress deadline to set for the deployment
+	ProgressDeadlineAnnotationKey = GroupName + "/progressDeadline"
```

Comment:
New annotations labels are `hyphen-separated`

```suggestion
	ProgressDeadlineAnnotationKey = GroupName + "/progress-deadline"
```

Reply:
Not sure why I had this flipped (heck, I think I may have even reviewed the PR that updated all of these...) :facepalm:

Reply:
We should also update the PR release note